.gitignore

```diff
@@ -16,3 +16,5 @@ vendor/re2/obj/libre2.a
 /bazel-*
 compile_commands.json
 .devbox
+.DS_Store
```

Comment:
I'm happy to take this change, but IMO this is the kind of thing that better lives in a [`core.excludesFile`](https://git-scm.com/docs/gitignore#_configuration) so that it is globally excluded on a developer's machine, instead of on a per-project basis. But since that feature is admittedly a bit esoteric, I'm happy to add it to the project so things "just work" a bit better for contributors on macs.

Reply:
TIL! I'll remove these changes in favor of `core.excludesFile`.